scripts/split.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-console */
+const execa = require('execa');
+const gitUrlParse = require('git-url-parse');
+const ora = require('ora');
+const path = require('path');
+const program = require('commander');
+const { find, kebabCase } = require('lodash');
+
+const getBranchUrl = async (branch) => {
+  const { stdout: remoteUrl } = await execa.shell('git config --get remote.origin.url');
+  const gitHttpsUrl = gitUrlParse(remoteUrl).toString('https');
+  return `git+${gitHttpsUrl}#${branch}`;
+};
+
+const getCurrentBranch = () => execa.shell('git rev-parse --abbrev-ref HEAD')
+  .then(({ stdout: branchName }) => branchName);
+
+const getPackageInfos = searchPackageName => execa.shell('lerna list -la --json')
+  .then(({ stdout: packageList }) => {
+    const searchPackage = find(
+      JSON.parse(packageList),
+      packageInformation => packageInformation.name === searchPackageName,
+    );
+    if (!searchPackage) {
+      return Promise.reject(new Error(`"${searchPackageName}" package not found`));
+    }
+    if (searchPackage.private) {
+      return Promise.reject(new Error(`"${searchPackage.name}" is a private package`));
+    }
+    return searchPackage;
+  });
+
+const getSubtreeBranch = (packageName, currentBranch) => {
+  const [scope, name] = packageName.split('/');
+  let leftPart = name;
```

Comment:
```suggestion
  const leftPart = name ? name : scope;
```

Reply:
```suggestion
  const leftPart = isString(name) ? name : scope;
```
or
```suggestion
  const leftPart =name || scope;
```
if you don't care about types